static/js/store.js

```diff
@@ -12,10 +12,10 @@ import {
 
 const syncConfig = {
   whitelist: [
-    "baselayer/SHOW_NOTIFICATION",
+    // "baselayer/SHOW_NOTIFICATION",
```

Comment:
Needed?

Reply:
Yes! This is to leverage the new shared tab data stuff. We can also share all the dispatch actions. The issue is, that if tab A had some error notifications, and you open tab B, they appear and don't go away until you click on them. A pain for some users, and not very useful anyway.